### ISSUE TITLE
It just completely stopped working

### ISSUE THREAD

**Issue by Stecks27** [edgescripts/Edgenuity-Scripts#4]
Idk what happened but the script stopped working.

**Comment by edgescripts** [comment]
> Idk what happened but the script stopped working.

Yeah, they updated egenuity so some scripts stopped working, message me on telegram at **+1 443-620-3964** and I'll explain more
